cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/TransformerConversionTest.java

```diff
@@ -521,26 +519,22 @@ private static T3xFlow threeWindingsTransformerFlow(ThreeWindingsTransformer twt
 
     private boolean sameFlow(T2xFlow expected, T2xFlow actual) {
         double tol = 0.00001;
-        if (Math.abs(expected.p1 - actual.p1) > tol ||
-            Math.abs(expected.q1 - actual.q1) > tol ||
-            Math.abs(expected.p2 - actual.p2) > tol ||
-            Math.abs(expected.q2 - actual.q2) > tol) {
-            return false;
-        }
-        return true;
+        // Comparison fails if actual has NaN values
+        return Math.abs(expected.p1 - actual.p1) < tol &&
+                Math.abs(expected.q1 - actual.q1) < tol &&
+                Math.abs(expected.p2 - actual.p2) < tol &&
+                Math.abs(expected.q2 - actual.q2) < tol;
```

Comment:
Shouldn't it be `<=` ?

Reply:
yes, although it is not really relevant here